### ISSUE TITLE
Fix bug

### ISSUE THREAD

**Issue by RFKryptic** [signalapp/Signal-iOS#4861]
- [ ] I have searched open and closed issues for duplicates
- [ ] I am submitting a bug report for existing functionality that does not work as intended
- [ ] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Describe here the issue that you are experiencing.

### Steps to reproduce
- using hyphens as bullet points
- list the steps
- that reproduce the bug

**Actual result:** Describe here what happens after you run the steps above (i.e. the buggy behaviour)

**Expected result:** Describe here what should happen after you run the steps above (i.e. what would be the correct behaviour)

### Device info

**Device**: iDevice X

**iOS version**: X.Y.Z

**Signal version:** Z.Y

**Comment by RFKryptic** [comment]
Someone hacked into my wallet. Seems to have been phishing through signal

**Comment by Genis-Sage** [comment]
Hey @RFKryptic
what exactly are you talking about?
what wallet are you talking about?
how is this related to Signal Messenger?
What did Signal do "wrong" so it caused you being hacked?

Wiith the currentiinformation , nnobody can do anything

**Comment by RFKryptic** [comment]
I think I figured it out. So from what I'm seeing I had a malware stuck to a file, it was pulling me in and out because I believe that it was stuck to a sound wave and anytime I did anything it would jump to something else. So, broke the walls down, renewed all my trust and I believe I got them out, but they still pop up even though everything's green. Also, I can't seem to figure out how to pull out my wallet. That has been a real problem. I need a little assistance with that lol.
